src/lib.rs

```diff
@@ -138,29 +142,34 @@
 //!     mono_font::{ascii::FONT_6X9, MonoTextStyle},
 //!     pixelcolor::BinaryColor,
 //!     prelude::*,
-//!     primitives::{Circle, Rectangle, Triangle, PrimitiveStyle},
+//!     primitives::{Circle, Rectangle, Triangle, PrimitiveStyle, StrokeAlignment},
 //!     text::Text,
 //!     mock_display::MockDisplay,
 //! };
 //!
 //! fn main() -> Result<(), std::convert::Infallible> {
 //!     // Create a new mock display
 //!     let mut display: MockDisplay<BinaryColor> = MockDisplay::new();
-//!     # display.set_allow_overdraw(true);
+//! #   display.set_allow_overdraw(true);
 //!
 //!     // Create styles used by the drawing operations.
 //!     let thin_stroke = PrimitiveStyle::with_stroke(BinaryColor::On, 1);
 //!     let thick_stroke = PrimitiveStyle::with_stroke(BinaryColor::On, 3);
+//!     let border_stroke = PrimitiveStyleBuilder::new()
+//!         .stroke_width(3)
+//!         .stroke_alignment(StrokeAlignment::Inside)
+//!         .stroke_color(BinaryColor::On)
+//!         .build();
```

Comment:
I like to keep the calls in the same order as the parameters in the `with_stroke` above, which makes it easier to spot the similarity between the statements IMO:

```suggestion
//!     let border_stroke = PrimitiveStyleBuilder::new()
//!         .stroke_color(BinaryColor::On)
//!         .stroke_width(3)
//!         .stroke_alignment(StrokeAlignment::Inside)
//!         .build();
```